src/OrbitSsh/Session.cpp

```diff
@@ -105,14 +81,11 @@ outcome::result<void> Session::MatchKnownHosts(const AddrAndPort& addr_and_port,
 
   const int check_result = libssh2_knownhost_checkp(
       known_hosts, addr_and_port.addr.c_str(), addr_and_port.port, fingerprint, fingerprint_length,
-      LIBSSH2_KNOWNHOST_TYPE_PLAIN | LIBSSH2_KNOWNHOST_KEYENC_RAW |
-          ((fingerprint_type + 1) << LIBSSH2_KNOWNHOST_KEY_SHIFT),
```

Comment:
This is an unrelated change that I just sneaked in. The flags parameter in `libssh2_knownhost_checkp` doesn't accept a fingerprint type. It just happened to not clash with other flags in the bitmask, that's why it worked in the first place.